readthedocs/rtd_tests/tests/test_sync_versions.py

```diff
@@ -105,12 +105,13 @@ def test_new_tag_update_active(self):
             self.pip.get_stable_version().identifier,
         )
 
-    def test_new_tag_update_inactive(self):
+    def test_new_tag_dont_update_inactive(self):
```

Comment:
Finally, I was able to fix and understand why this test was failing, first this test is wrong, why? Because our current logic says (and do)

https://github.com/rtfd/readthedocs.org/blob/4031b9078efba9948301e89c948a236671086d26/readthedocs/restapi/views/model_views.py#L176-L177

why the test was passing?

Because the version was created with `type='unknown'`, so it was being skipped here

 https://github.com/rtfd/readthedocs.org/blob/4031b9078efba9948301e89c948a236671086d26/readthedocs/projects/version_handling.py#L238-L240

Why the test fail now?

Because we are filtering by branches and tags only, so the unknown version was being deleted.

Reply:
Heh, good catch!